build-system/babel-config/pre-closure-config.js

```diff
@@ -98,6 +98,7 @@ function getPreClosureConfig() {
     plugins: preClosurePlugins,
     presets: preClosurePresets,
     retainLines: true,
+    sourceMaps: true,
```

Comment:
FMI: why was this false before?

Reply:
Good question. It was false because we were using `gulp-sourcemaps` which was separately collecting and writing sourcemap info to file. Now we rely on Babel to do its part.